packaging/CMakeLists.txt

```diff
@@ -0,0 +1,144 @@
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+
+##
+# Custom locations
+##
+
+set(HALIDE_INSTALL_CMAKEDIR
+    "${CMAKE_INSTALL_LIBDIR}/cmake/Halide"
+    CACHE STRING "Path to Halide cmake files")
+
+##
+# Main library exports
+##
+
+# TODO: fix when autoschedulers are refactored
```

Comment:
Are there tracking issues for the new TODOs inserted? If not, now is probably a good time to add them (and add the issue URL as a comment)

Reply:
There are not, yet. I'll have to go through and make those.